pilot/pkg/model/xds_cache_test.go

```diff
@@ -0,0 +1,424 @@
+package model
```

Comment:
Can you please move these tests here https://github.com/istio/istio/blob/master/pilot/pkg/xds/xds_cache_test.go? or did you put them here for any specific reason?

Reply:
I put them here because I need to be in the same package and access the indexes which are internal fields